fluent-release/yum/fluent-release.spec.in

```diff
@@ -0,0 +1,68 @@
+# -*- rpm -*-
+
+%define _amzn %{?amzn:%{amzn}}%{!?amzn:0}
+%define is_amazon_linux (%{_amzn} != 0)
+
+Name:		@PACKAGE@
+Version:	@VERSION@
+Release:	@RELEASE@%{?dist}
+Summary:	Fluent Package release files
+BuildArchitectures: noarch
+
+License:	ASL 2.0
+URL:		https://docs.fluentd.org/
+Source0:	@PACKAGE@-@VERSION@.tar.gz
+
+%if ! %{is_amazon_linux}
+Requires: epel-release
+Requires: dnf-command(config-manager)
+%else
+%if 0%{_amzn} == 2
```

Comment:
```suggestion
%if %{_amzn} == 2
```

Do we need `0`?
It looks to me that `_amzn` should have some values because `%{!?amzn:0}` is applied in advance if `amzn` is not defined.